hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/processors/ProcessorModule.java

```diff
@@ -27,7 +27,13 @@
 import org.hyperledger.besu.evm.contractvalidation.MaxCodeSizeRule;
 import org.hyperledger.besu.evm.contractvalidation.PrefixCodeRule;
 
-@Module(includes = {HtsTranslatorsModule.class, HasTranslatorsModule.class, HssTranslatorsModule.class})
+@Module(
+        includes = {
+            HtsTranslatorsModule.class,
+            Hts0x16cTranslatorsModule.class,
```

Comment:
Can revert this change. (Presumably unintentional.)

Reply:
Reverted

Reply:
Fixed